Src/FluentAssertions/Equivalency/CollectionMemberAssertionRuleDecorator.cs

```diff
@@ -26,17 +26,18 @@ public bool CanHandle(IEquivalencyValidationContext context, IEquivalencyAsserti
 
         private static EquivalencyValidationContext CreateAdjustedCopy(IEquivalencyValidationContext context)
         {
-            return new EquivalencyValidationContext
+            var adjustedCopy = new EquivalencyValidationContext
             {
                 CompileTimeType = context.CompileTimeType,
                 Expectation = context.Expectation,
-                SelectedMemberDescription = context.SelectedMemberDescription,
-                SelectedMemberInfo = context.SelectedMemberInfo,
-                SelectedMemberPath = CollectionMemberMemberInfo.GetAdjustedPropertyPath(context.SelectedMemberPath),
-                Because = context.Because,
-                BecauseArgs = context.BecauseArgs,
+                SelectedMember = context.SelectedMember.Clone(),
+                Reason =  context.Reason,
                 Subject = context.Subject
             };
+
+            adjustedCopy.SelectedMember.Path = CollectionMemberMemberInfo.GetAdjustedPropertyPath(context.SelectedMember.Path);
```

Comment:
Should the `Description` also still be set?

Reply:
Ah yes.